### DISCUSSION TITLE
Reference/Addendum Pages

### DISCUSSION THREAD

**Discussion by shimizurei** [mealie-recipes/mealie#802] [Feature Request]
**Before submitting this feature request I have**

- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**

Reference pages. Since there is already a custom page option, what if you could create a blank page and for example, make a note of common substitutions? For example, when I run out of eggs, I like to substitute applesauce in for my eggs. There are a ton of possible substitutes for many ingredient parts and it would be nice to collect them in one place.

**Suggest A Solution**

The page function doesn't have to be that detailed. Just create a blank, custom page and using Markdown, add your info. Then have a "References" or "Addendum" page that collects all these little note/reference pages into one. Then the page can be added to the sidebar.

**Comment by hay-kot** [comment]
This seems like something I would use notes for. I'd rather have the substitutions in the recipe rather than a global location. 

I don't think this warrants it's own page. I haven't found substitutions to be that universal either.

**Comment by michael-genson** [comment]
Mealie has changed drastically since this discussion was opened, so I'm closing it since it's no longer relevant (we no longer have custom pages). Notes can be used to achieve this.
